games-util/pyfa/pyfa-2.9.4.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+PYTHON_COMPAT=( python3_6 )
+PYTHON_REQ_USE="sqlite,threads"
+
+inherit desktop eutils gnome2-utils python-r1
+
+DESCRIPTION="Python Fitting Assistant - a ship fitting application for EVE Online"
+HOMEPAGE="https://github.com/pyfa-org/Pyfa"
+
+RESTRICT="mirror bindist"
+LICENSE="GPL-3+ all-rights-reserved"
+SLOT="0"
+if [[ ${PV} = 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/pyfa-org/Pyfa.git"
+	inherit git-r3
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/pyfa-org/Pyfa/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+IUSE="+graph"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	>=dev-python/logbook-1.0.0[${PYTHON_USEDEP}]
+	dev-python/python-dateutil[${PYTHON_USEDEP}]
+	>=dev-python/requests-2.0.0[${PYTHON_USEDEP}]
+	>=dev-python/sqlalchemy-1.3.0[${PYTHON_USEDEP}]
+	>=dev-python/wxpython-4.0.4[webkit,${PYTHON_USEDEP}]
+	>=dev-python/cryptography-2.3[${PYTHON_USEDEP}]
+	>=dev-python/roman-2.0.0[${PYTHON_USEDEP}]
+	>=dev-python/markdown2-2.3.0[${PYTHON_USEDEP}]
+	>=dev-python/beautifulsoup-4.6.0[${PYTHON_USEDEP}]
+	>=dev-python/pyyaml-5.1[${PYTHON_USEDEP}]
+	graph? (
+		>=dev-python/matplotlib-2.0.0[wxwidgets,${PYTHON_USEDEP}]
+		dev-python/numpy[${PYTHON_USEDEP}] )
+	${PYTHON_DEPS}"
+DEPEND="app-arch/zip"
+
+[[ ${PV} = 9999 ]] || S=${WORKDIR}/Pyfa-${PV}
+
+src_prepare() {
+	# get rid of CRLF line endings introduced in 1.1.10 so patches work
+	edos2unix config.py pyfa.py gui/bitmap_loader.py service/settings.py
+
+	# load gameDB and images from separate staticdata directory
+	eapply "${FILESDIR}/${PN}-2.9.3-staticdata.patch"
```

Comment:
You could use `PATCHES=()` array, and call `default` after `edos2unix` instead of eapply_user. It's a style thing, use whatever you like, but pointing out another possibility.